oauthproxy.go

```diff
@@ -908,13 +893,12 @@ func (p *OAuthProxy) Proxy(rw http.ResponseWriter, req *http.Request) {
 		}
 
 	case ErrAccessDenied:
-		p.ErrorPage(rw, req, http.StatusUnauthorized, "Permission Denied", "Unauthorized")
+		p.ErrorPage(rw, req, http.StatusUnauthorized, "Unauthorized")
```

Comment:
I think this should mirror line 814 and be `http.StatusForbidden` (the underlying function returns this error in `!authorized` cases)

Reply:
ToDo: Make this consistent, should be `http.StatusForbidden`